pkg/component/component.go

```diff
@@ -131,6 +132,25 @@ func CreateFromPath(client *occlient.Client, name string, componentImageType str
 	return nil
 }
 
+// Reads file path form URL file:///C:/path/to/file to C:\path\to\file
+func ReadFilePath(u *url.URL) string {
```

Comment:
this functions  `ReadFilePath` and `GenFilePath` should go in `pkg/util/util.go`